controllers/admin/AdminModulesController.php

```diff
@@ -272,10 +272,13 @@ public function ajaxProcessGetTabModulesList()
         if ($back) {
             $back .= '&tab_modules_open=1';
         }
-        $modules_list = array('installed' =>array(), 'not_installed' => array());
         if ($tab_modules_list) {
             $tab_modules_list = explode(',', $tab_modules_list);
-            $modules_list_unsort = $this->getModulesByInstallation($tab_modules_list);
+            if ($admin_list_from_source = Tools::getValue('admin_list_from_source')) {
```

Comment:
You can directly pass the return value of Tools::getValue() without any condition